kitchen.dokken.yml

```diff
@@ -35,29 +35,19 @@ platforms:
     driver:
       image: dokken/debian-8
       pid_one_command: /bin/systemd
-      intermediate_instructions:
-        - RUN /usr/bin/apt-get update
```

Comment:
I'm pretty sure this is still needed?

Reply:
Seem to be happy without it but I can put it back it. It definitely is needed for certain repos.